File: Sorting/python/05_수정렬하기3.py
# fast campus 강의
# https://www.acmicpc.net/problem/10989
# 1

import sys

N = int(sys.stdin.readline())

array = [0] * 10001

for _ in range(N):
    temp = int(sys.stdin.readline())
    array[temp] += 1

for i in range(10001):
    j = array[i]
    for _ in range(j):
        print(i)

File: Sorting/c++/05_수정렬하기3.cpp
// fast campus 강의
// https://www.acmicpc.net/problem/10989
// 2

#include <iostream>

using namespace std;

int N;
int arr[10001];

int main()
{
    scanf("%d", &N);
    for (int i = 0; i < N; i++) {
        int tmp;
        scanf("%d", &tmp);
        arr[tmp]++;
    }

    for (int i = 0; i < 10001; i++) {
        for (int j = 0; j < arr[i]; j++)
            printf("%d\n", i);
    }
}